src/UIKit/UIEnums.cs

```diff
@@ -1125,9 +1137,14 @@ public enum UITextDirection : long {
 	}
 	
 	// NSInteger -> UITextInput.h
+	// NSWritingDirection == UITextWritingDirection
 	[Native]
 	[NoWatch]
+#if XAMCORE_4_0
+	public enum NSWritingDirection : long {
+#else
 	public enum UITextWritingDirection : long {
+#endif
```

Comment:
This is becoming confusing
```
src/AppKit/Enums.cs:    public enum NSWritingDirection : long {
src/Foundation/Enum.cs: [Obsolete ("Use NSWritingDirection in AppKit instead.")]
src/Foundation/Enum.cs: public enum NSWritingDirection : long {
```
and, for us, it's really different types (if some new, shared macOS+iOS API, start using them)

Reply:
Oh woow nice catch @spouliot will expose AppKit's into xkit.cs and use that instead 👍

Reply:
Fixed in https://github.com/xamarin/xamarin-macios/pull/6553/commits/692b68bae277a1f1474d28e5d4535b39c6122704